app/controllers/investigations/test_results_controller.rb

```diff
@@ -60,18 +59,15 @@ def update
       return render :edit
     end
 
-    result = UpdateTestResult.call(
+    UpdateTestResult.call(
```

Comment:
```suggestion
    UpdateTestResult.call!(
```
Since we removed the `success` conditional. Throws exception on failure, otherwise we will not know if it failed and may erroneously report to the user that it succeeded.